media/js/base/mozilla-utils.js

```diff
@@ -12,31 +12,8 @@ if (typeof Mozilla === 'undefined') {
 
     var Utils = {};
 
-    /**
-     * Bug 393263 A special function for IE < 9.
-     * Without this hack there is no prompt to download after they click. sigh.
-     * @param {link} direct link to download URL
-     * @param {userAgent} optional UA string for testing purposes.
-     */
-    Utils.triggerIEDownload = function(link, userAgent) {
-        'use strict';
-        var ua = userAgent !== undefined ? userAgent : navigator.userAgent;
-        // Only open if we got a link and this is IE < 9.
-        if (link && window.site.platform === 'windows' && /MSIE\s[1-8]\./.test(ua)) {
-            window.open(link, 'download_window', 'toolbar=0,location=no,directories=0,status=0,scrollbars=0,resizeable=0,width=1,height=1,top=0,left=0');
-            window.focus();
-        }
-    };
-
-    // attach an event to all the download buttons to trigger the special
-    // ie functionality if on ie
+    // a11y assistance
     Utils.initDownloadLinks = function() {
-        $('.download-link').each(function() {
-            var $el = $(this);
-            $el.click(function() {
-                Utils.triggerIEDownload($el.data('direct-link'));
-            });
-        });
         $('.download-list').attr('role', 'presentation');
```

Comment:
I know a11y is important, but it looks like this was added a long time ago. Is this `role` still useful enough on its own to keep this function around?

Reply:
It's a good question that I didn't really spend much time thinking about. (I was more focused on keeping scope for this PR as limited as possible.)

**I am not an a11y expert!**

That being said, I believe the intent here is to keep screen readers from treating the `<ul>` as an actual list to be iterated over, as the visitor *should* only care about the one `<li>` displayed. However, all other `<li>`s are set to `display: none;`, which (I believe) should prevent them from being recognized by screen readers.

Unless you disagree with the above assessment, I'll remove this function. Let me know!

Reply:
Remove away 👍